src/util.ts

```diff
@@ -49,10 +49,13 @@ export function errToString(error: any): string {
 	return error.toString();
 }
 
-export function showResult(result: string, column?: vscode.ViewColumn): Thenable<void> {
+export function showResult(result: string, filename: string, column?: vscode.ViewColumn): Thenable<void> {
```

Comment:
Change signature since we now show documents as document.json, and scrapbook results as result.json.